api/Element.json

```diff
@@ -2531,49 +2795,107 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/querySelectorAll",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
             },
             "chrome": {
-              "version_added": null
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": true
             },
             "firefox": {
-              "version_added": null
+              "version_added": "3.5"
             },
             "firefox_android": {
-              "version_added": null
-            },
-            "ie": {
-              "version_added": null
+              "version_added": true
             },
+            "ie": [
+              {
+                "version_added": "9"
+              },
+              {
+                "version_added": "8",
+                "partial_implementation": true,
+                "notes": "<code>querySelectorAll()</code> is supported, but only for CSS 2.1 selectors."
+              }
+            ],
             "opera": {
-              "version_added": null
+              "version_added": "10"
             },
             "opera_android": {
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "3.2"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "scope_pseudo_class": {
```

Comment:
This is in the article's table, but I don't think it should be? Can I remove it? https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll#Browser_compatibility

Reply:
Yes, scope is obsolete anyways.